README.md

```diff
@@ -80,16 +80,20 @@ Getting Started on AWS
 - Get firectl binary:
 
   ```
-  curl -Lo firectl https://firectl-release.s3.amazonaws.com/firectl-v0.1.0
-  curl -Lo firectl.sha256 https://firectl-release.s3.amazonaws.com/firectl-v0.1.0.sha256
+  curl -Lo firectl-v0.2.0 https://github.com/firecracker-microvm/firectl/releases/download/v0.2.0/firectl-v0.2.0
+  curl -Lo firectl.sha256 https://github.com/firecracker-microvm/firectl/releases/download/v0.2.0/firectl-v0.2.0.sha256
   sha256sum -c firectl.sha256
+  mv firectl-v0.2.0 firectl
   chmod +x firectl
   ```
 
 - Get Firecracker binary:
 
   ```
-  curl -Lo firecracker https://github.com/firecracker-microvm/firecracker/releases/download/v0.16.0/firecracker-v0.16.0
+  curl -Lo firecracker.tgz https://github.com/firecracker-microvm/firecracker/releases/download/v1.1.2/firecracker-v1.1.2-x86_64.tgz
+  tar -xvf firecracker.tgz
+  mv release-v1.1.2-x86_64/firecracker-v1.1.2-x86_64 firecracker
+  rm -rf firecracker.tgz release-v1.1.2-x86_64
```

Comment:
Personally I feel like deleting the zip and the unzipped content is not necessary in the README.